packages/back-office-subscribers/nsip-exam-timetable/index.js

```diff
@@ -0,0 +1,66 @@
+const { prismaClient } = require('../lib/prisma');
+
+module.exports = async (context, message) => {
+	context.log(`invoking nsip-exam-timetable function with message: ${JSON.stringify(message)}`);
```

Comment:
As we revisit each of these functions, I think it would be good remove these `context.log`s that log the message body. Some will contain PII so best to not log the whole message